.github/workflows/test.yaml

```diff
@@ -335,7 +333,7 @@ jobs:
       - name: use node ${{ matrix.node }}
         uses: actions/setup-node@v2
         with:
-          node-version: 10
+          node-version: ${{ matrix.node }}
```

Comment:
there is no matrix with node versions in this job, the name is just a bug here I would say